event/event.yaml

```diff
@@ -3,71 +3,55 @@ components:
     Event:
       description: |-
         The optional container for event configuration.
+        To get cp-dp data both cp_events and dp_events must be explicitly enabled. 
       type: object
       properties:
-        enable:
+        cp_events:
           description: |-
-            True to enable all events. 
-            Enabling this option may affect the resultant packet payload due to 
-            additional instrumentation data.
+            The optional container for control plane(cp) event configuration.
+          $ref: '#/components/schemas/Event.CPEvents'
           type: boolean
           default: false
           x-field-uid: 1
-        link:
-          $ref: '#/components/schemas/Event.Link'
-          x-field-uid: 2
-        rx_rate_threshold:
-          $ref: '#/components/schemas/Event.RxRateThreshold'
-          x-field-uid: 3
-        route_advertise_withdraw:
-          $ref: '#/components/schemas/Event.RouteAdvertiseWithdraw'
-          x-field-uid: 4
-    Event.RxRateThreshold:
-      description: |-
-        The optional container for rx rate threshold event configuration.
-      type: object
-      properties:
-        enable:
+        dp_events:
           description: |-
-            True to enable the rx_rate_threshold event. 
+            The optional container for data plane(dp) event configuration.
```

Comment:
Lets remove optional for now since enabling feature requires this container to be included.